### PR TITLE
refactor(ipfs): endpoint and flag Scaleway CLI Labs

### PR DESCRIPTION
**Pull request by OnsagerHe**
### Your checklist for this pull request

- [X] Check that the commit messages match our requested structure.
- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: labs/ipfs/api-cli/ipfs-operations.mdx [hunk 1]
```diff
@@ -39,13 +39,15 @@ This operation allows you to add content by its CID in a volume.
   It suppose that the content pointed by the CID must already be present on the public IPFS network.
 </Message>
 
+<Message type="important">
+    To use the ipfs command in the Scaleway CLI, set the SCW_ENABLE_LABS=true flag.
```
Comment (ldecarvalho-doc) [inline labs/ipfs/api-cli/ipfs-operations.mdx:43]:
```suggestion
    To manage IPFS with the Scaleway CLI, you must set the `SCW_ENABLE_LABS` flag to `true`, as such: `SCW_ENABLE_LABS=true`
```

File: labs/ipfs/api-cli/ipfs-operations.mdx [hunk 2]
```diff
@@ -39,13 +39,15 @@ This operation allows you to add content by its CID in a volume.
...
+    It's advisable to export it as an environment variable, so you don't have to set it for every command.
```
Comment (ldecarvalho-doc) [inline labs/ipfs/api-cli/ipfs-operations.mdx:44]:
```suggestion
    We recommend you export the flag as an environment variable, so you do not have to set it for every command.
```

File: labs/ipfs/api-cli/volumes-operations.mdx [hunk 3]
```diff
@@ -92,18 +92,23 @@ The [Scaleway Command Line Interface (CLI)](https://github.com/scaleway/scaleway
 
 ## Create a volume
 
+<Message type="important">
+    To use the ipfs command in the Scaleway CLI, set the SCW_ENABLE_LABS=true flag.
+    It's advisable to export it as an environment variable, so you don't have to set it for every command.
```
Comment (ldecarvalho-doc) [inline labs/ipfs/api-cli/volumes-operations.mdx:97]: pareil que les suggestions d'avant

File: labs/ipfs/quickstart.mdx [hunk 4]
```diff
@@ -68,6 +68,11 @@ Now that we have a volume, we can pin content from CIDs (content identifier) and
   It suppose that the content pointed by the CID must already be present on the public IPFS network.
 </Message>
 
+<Message type="important">
+    To use the ipfs command in the Scaleway CLI, set the SCW_ENABLE_LABS=true flag.
+    It's advisable to export it as an environment variable, so you don't have to set it for every command.
```
Comment (ldecarvalho-doc) [inline labs/ipfs/quickstart.mdx:73]: same here ;)

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

**Pull request merged**